src/dash-playlist-loader.js

```diff
@@ -727,15 +727,12 @@ export default class DashPlaylistLoader extends EventTarget {
       mpl.createMupOnMedia_ = null;
     }
 
-    // clear any pending timeouts, master was just updated
-    // so if minimumUpdatePeriodTimeout_ is still valid
-    window.clearTimeout(mpl.minimumUpdatePeriodTimeout_);
-    mpl.minimumUpdatePeriodTimeout_ = null;
-    mpl.createMUPTimeout_();
-  }
+    // clear any pending timeouts
+    if (mpl.minimumUpdatePeriodTimeout_) {
+      window.clearTimeout(mpl.minimumUpdatePeriodTimeout_);
+      mpl.minimumUpdatePeriodTimeout_ = null;
+    }
 
-  createMUPTimeout_() {
-    const mpl = this.masterPlaylistLoader_;
     let mup = mpl.master && mpl.master.minimumUpdatePeriod;
 
     // If the minimumUpdatePeriod has a value of 0, that indicates that the current
```

Comment:
It looks like a 0 value is valid for instantaneous updates
> The MPD@minimumUpdatePeriod is present but is recommended to be set to 0 in order to announce instantaneous segment updates.

Should we immediately request instead of waiting for `targetDuration`?

Reply:
From what I understand `instantaneous` segment updates will only be released after waiting for the target duration of the last segment in the playlist to expire. So if the last segment is `X` with a target duration of 2s then we need to wait 2s from the current update for the next segment `Z` to be added.